test_runner/regress/test_pageserver_secondary.py

```diff
@@ -903,14 +904,17 @@ def test_migration_to_cold_secondary(neon_env_builder: NeonEnvBuilder):
         remote_storage_kind=RemoteStorageKind.MOCK_S3,
     )
 
+    tenant_conf = TENANT_CONF
+    tenant_conf["heatmap_period"] = "0s"
```

Comment:
I think this will modify `TENANT_CONF`, need to deep-copy it (python footgun...)

Reply:
https://github.com/neondatabase/neon/pull/10895/commits/cd29fcd5c055a34e7ea8ed50aeeef5c64a4aa3d9